helpdesk_mgmt_ticket_invoice/models/account_move.py

```diff
@@ -0,0 +1,28 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    invoice_ticket_ids = fields.Many2many("helpdesk.ticket", relation="account_helpdesk")
+
+    def action_view_tickets(self):
+        return {
```

Comment:
This method will always open the tickets in tree view, it would be good if there was only one ticket it would open in form and if not in tree